pkgs/shells/fish/default.nix

```diff
@@ -139,13 +139,38 @@ let
       # the shell's actual version (and what it displays when running `fish
       # --version`), as well as the local documentation for all builtins (and
       # maybe other things).
-      url = "https://github.com/fish-shell/fish-shell/releases/download/${version}/${pname}-${version}.tar.gz";
-      sha256 = "1vblmb3x2k2cb0db5jdyflppnlqsm7i6jjaidyhmvaaw7ch2gffm";
+      url = "https://github.com/fish-shell/fish-shell/releases/download/${version}/${pname}-${version}.tar.xz";
+      sha256 = "sha256-TwKT7Z9qa3fkfUHvq+YvMxnobvyL+DzFhzMET7xvkhE=";
     };
 
-    # We don't have access to the codesign executable, so we patch this out.
-    # For more information, see: https://github.com/fish-shell/fish-shell/issues/6952
-    patches = lib.optional stdenv.isDarwin ./dont-codesign-on-mac.diff;
+    # Fix FHS paths in tests
+    postPatch = ''
```

Comment:
Can this also go in 6944e359dc35bf6c0b80a358c4601d2ead852a18?

Reply:
Ups, got confused in the rebase. Fixed, thanks for noticing!

Reply:
> I just didn't want to have to wait for that `TODO` to be finished before getting this version bump in. However, it seems like your recent push removed that, so I'm fine with merging that in with this PR (assuming it works as expected). Any future test improvements can always be done in a separate PR :)

Agreed. Pushed once again but seems like the build runs fine on `aarch64-linux` and `x86_64-linux`. Also macOS works on my machine.